### ISSUE TITLE
Setting global variables doesn't persist into Postman app

### ISSUE THREAD

**Issue by lukeselbeck** [postmanlabs/newman#2285]
1. Newman Version (can be found via `newman -v`): 4.6.0
 2. OS details (type, version, and architecture): MacOs v10.14.6
 3. Are you using Newman as a library, or via the CLI? CLI
 3. Did you encounter this recently, or has this bug always been there:  Just installed newman today
 4. Expected behaviour: when newman is run with the `--global-var` option, the modified global var should be visible in the Postman app global variables
 5. Command / script used to run Newman: `newman run ./my.json --global-var "myGlobalVar=foo"`
 6. Sample collection, and auxiliary files (minus the sensitive details): 
```
{
	"info": {
		"_postman_id": "c890745c-6220-40e6-8fd3-b5d971aad986",
		"name": "startup",
		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
	},
	"item": [
		{
			"name": "global vars",
			"event": [],
			"request": {
				"method": "GET",
				"header": [],
				"url": {
					"raw": "https://httpstat.us/200",
					"protocol": "https",
					"host": [
						"httpstat",
						"us"
					],
					"path": [
						"200"
					]
				},
				"description": "updates global vars"
			},
			"response": []
		}
	],
	"protocolProfileBehavior": {}
}
```

7. Actual Behavior:  the global variable is useable during the newman run, but opening up the postman app/gui global variables pane shows that no global variable has been set.

Basically, I want to be able to set postman app global variables from the command line, and I would have expected `--global-var` to do that.  There is no documentation that says the global var set during the newman run only exists for the duration of the newman run.

**Comment by jimc404** [comment]
Global variables are set during the time when `newman` runs. Once newman exits all of it's runtime data (stored in RAM) is lost, global vars for the newman run only exist in the scope of the execution and are hence lost. Postman runs in a different process from newman and naturally can't access it's runtime data.

Collections share data across postman infrastructure but since newman does not alter the collections, variable data from newman can't be exported outside.

**Comment by jimc404** [comment]
> There is no documentation that says the global var set during the newman run only exists for the duration of the newman run.

Since `--global-var` here works similar to environment variables it's probably implicitly understood that  global vars will be lost after execution. After all a child process can't alter the environment data of it's parent process, the runtime data of a `newman run` won't be carried upwards.

**Comment by lukeselbeck** [comment]
Understood.  If not through newman, do you know of any way to modify postman app variables programmatically, with the command line or otherwise?

**Comment by jimc404** [comment]
I haven't really tweaked around with the variables enough to give a good answer.

My best guess would be to use an environment on the postman cloud since it can be accessed from anywhere with just it's ID.

You can open a new issue to discuss this :)
